Userland/Applications/PixelPaint/ImageEditor.cpp

```diff
@@ -313,6 +336,11 @@ void ImageEditor::mousedown_event(GUI::MouseEvent& event)
         return;
     }
 
+    if (m_select_color_mode && !m_active_tool->is_overriding_alt()) {
+        set_editor_color_to_color_at_mouse_position(event);
+        return; // Do not function normally when in select_color_mode.
```

Comment:
This comment is a bit vague/confusing to me.
You don't need to change it but does: 
`// Pick color instead of activating active tool when holding alt`
cover what you want to say here?